doc/api/tls.md

```diff
@@ -851,12 +851,12 @@ publicly trusted list of CAs as given in
   * `passphrase` {string} A string containing the passphrase for the private
     key or pfx.
   * `cert` {string|string[]|Buffer|Buffer[]} A string, `Buffer`, array of
-    strings, or array of `Buffer`s containing the certificate key of the server
-    in PEM format. (Required)
+    strings, or array of `Buffer`s containing the server's certificate chain
+    (server's certificate and intermediates) in PEM format. (Required)
```

Comment:
a) Why is it preferable to place them in `ca`?  b) even if it is preferable, the ability to provide an entire cert chain here, and the order they should be in, must be documented and it isn't c) I'm about to rewrite much of this, the docs are in rough shape, and this is on my list of things to fix